src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -35,6 +35,8 @@ public class ConnectionService
         private const string SqlAzureEdition = "SQL Azure";
         public const int MaxTolerance = 2 * 60; // two minutes - standard tolerance across ADS for AAD tokens
 
+        public const int MaxTries = 10; // Max number of tries to wait for a serverless database to start up when its paused before giving up.

```

Comment:
Where is 10 from? That seems like a large number - typically retries are only 3 - 5 times. Especially if there's a timeout delay for each attempt.

Reply:
Randomly picked, just to ensure things work, I'm in the process of trying to figure out what's the minimum amount of tries required, I'll lower it to 5 for now.

Reply:
Serverless DBs generally resume in ~ 45 seconds.. so 2 retries should be enough, but we can also ceil at 3 or 5 as we want to choose..